### ISSUE TITLE
Scaling seems doesn't work for python

### ISSUE THREAD

**Issue by liu-qingzhen** [MIT-SPARK/TEASER-plusplus#117]
Hi, TEASER++ works great for same scale point clouds with large outliers.  But for point clouds with large scaling difference, the result seems not very good. I'm not sure if it is because the Python binding does not support scaling? I tuned the parameter estimate_scaling in helpers.py to be True, but the performance seems unchanged. 
<img width="743" alt="image" src="https://user-images.githubusercontent.com/70200053/135430287-4dc77a70-ac46-4149-a7f4-b20b2a15852b.png">

**Comment by liu-qingzhen** [comment]
The result of the model will return the scale. I forgot the code but the scale can be derived from result.s as well as the rotation and translation (result.T).

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ttsesm** [comment]
Sure it returns an `s` value, however it is not correct. When the scaling ratio between the two pcds is quite big, it fails to address the registration and the scaling value doesn't really help much.

In any case, thank you for your time.

**Comment by liu-qingzhen** [comment]
Yeah, they said it is because of the feature extraction with fpfh is not accurate. The scale estimation model makes sense, but the performance is not good with traditional feature descriptor.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ttsesm** [comment]
I see, did you test with an alternative feature descriptor, other than the fpfh, to see whether it performs better?

**Comment by jingnanshi** [comment]
@ttsesm FPFH is not designed to be scale-invariant (see the original paper). Maybe try this: https://link.springer.com/article/10.1007/s00521-017-2964-1?

**Comment by nkemter** [comment]
Is this bug/problem specific to Python or does this also occur with MATLAB and C++ implementations?

**Comment by jingnanshi** [comment]
@nkemter This is not TEASER++'s issue, but FPFH's. You can use scale-invariant features, match them and TEASER++ should work.
